internal/models/users/users.go

```diff
@@ -43,6 +43,12 @@ type UserSessionData struct {
 	Email string
 }
 
+type UserTableRow struct {
+	Username  string
+	Score     int
+	Placement int
+}
+
```

Comment:
think similar struct already exists in user ranking model

Reply:
unused code, removed